crates/dojo-indexer/src/graphql/entity.rs

```diff
@@ -57,12 +90,75 @@ pub async fn entities(context: &Context) -> FieldResult<Vec<Entity>> {
     let entities = sqlx::query_as!(
         Entity,
         r#"
-            SELECT * FROM entities
+            SELECT 
+                id,
+                name,
+                partition_id,
+                partition_keys,
+                transaction_hash,
+                created_at as "created_at: _",
+                updated_at as "updated_at: _"
+            FROM entities
         "#
     )
     .fetch_all(&mut conn)
-    .await
-    .unwrap();
+    .await?;
+
+    Ok(entities)
+}
+
+pub async fn entities_by_partition_id(
+    context: &Context,
+    partition_id: String,
+) -> FieldResult<Vec<Entity>> {
+    let mut conn = context.pool.acquire().await.unwrap();
```

Comment:
Is it safe to unwrap like this? Or should we do some error handling?

Reply:
Ah yep, will propagate the error here